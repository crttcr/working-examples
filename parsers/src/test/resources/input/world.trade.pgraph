// World Trade models international trade flow
//
//
// To add:
// TARIFF
// IMPORT
// EXPORT
//
// COUNTRY + PRODUCT as separate models
// Combine and add trade relations
//
// Apply trade data as time-based animation
//

## vertex.auto.id = "GENERATE_SEQUENCE"

(cyp: COUNTRY { name = "Cyprus"})
(grc: COUNTRY { name = "Greece"})
(gbr: COUNTRY { name = "United Kingdom"})
(lby: COUNTRY { name = "Libya"})
(isr: COUNTRY { name = "Israel"})
(ita: COUNTRY { name = "Italy"})
(fra: COUNTRY { name = "France"})
(usa: COUNTRY { name = "United States"})



(animal: PRODUCT_CLASS { class = "Animal"})
(vegetable: PRODUCT_CLASS { class = "Vegetable"})
(food: PRODUCT_CLASS { class = "Food"})
(drug: PRODUCT_CLASS { class = "Drugs"})
(mineral: PRODUCT_CLASS { class = "Minerals"})
(chemical: PRODUCT_CLASS { class = "Chemicals"})
(fuel: PRODUCT_CLASS { class = "Fuel"})
(hides_skins: PRODUCT_CLASS { class = "Hides/Skins"})
(textiles_clothes: PRODUCT_CLASS { class = "Textiles/Clothes"})
(wood: PRODUCT_CLASS { class = "Wood"})
(stone_glass: PRODUCT_CLASS { class = "Stone/Glass"})

(isr)--[:EXPORTS]->(drug)
(isr)--[:IMPORTS]->(drug)