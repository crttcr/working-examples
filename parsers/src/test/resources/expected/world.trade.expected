## vertex.auto.id = "GENERATE_SEQUENCE"
VERTEX: cyp
	LABEL: COUNTRY
		PROP name = "Cyprus"
VERTEX: grc
	LABEL: COUNTRY
		PROP name = "Greece"
VERTEX: gbr
	LABEL: COUNTRY
		PROP name = "United Kingdom"
VERTEX: lby
	LABEL: COUNTRY
		PROP name = "Libya"
VERTEX: isr
	LABEL: COUNTRY
		PROP name = "Israel"
VERTEX: ita
	LABEL: COUNTRY
		PROP name = "Italy"
VERTEX: fra
	LABEL: COUNTRY
		PROP name = "France"
VERTEX: usa
	LABEL: COUNTRY
		PROP name = "United States"
VERTEX: animal
	LABEL: PRODUCT_CLASS
		PROP class = "Animal"
VERTEX: vegetable
	LABEL: PRODUCT_CLASS
		PROP class = "Vegetable"
VERTEX: food
	LABEL: PRODUCT_CLASS
		PROP class = "Food"
VERTEX: drug
	LABEL: PRODUCT_CLASS
		PROP class = "Drugs"
VERTEX: mineral
	LABEL: PRODUCT_CLASS
		PROP class = "Minerals"
VERTEX: chemical
	LABEL: PRODUCT_CLASS
		PROP class = "Chemicals"
VERTEX: fuel
	LABEL: PRODUCT_CLASS
		PROP class = "Fuel"
VERTEX: hides_skins
	LABEL: PRODUCT_CLASS
		PROP class = "Hides/Skins"
VERTEX: textiles_clothes
	LABEL: PRODUCT_CLASS
		PROP class = "Textiles/Clothes"
VERTEX: wood
	LABEL: PRODUCT_CLASS
		PROP class = "Wood"
VERTEX: stone_glass
	LABEL: PRODUCT_CLASS
		PROP class = "Stone/Glass"
EDGE: EXPORTS
EDGE: IMPORTS